README.md

```diff
@@ -52,6 +52,15 @@ Slideshow is https://github.com/sachinchoolur/lightslider. Just copied the dist
 ## Local development
 
 Running the app locally requires:
+
+Note: 
+- The postgres version on your local environment might be different from the one on the
+        docker-compose.yml file. 
+        They both have to be of the version before running the docker command, to avoid running 
+        into postgis error.
+- If you get an error that the port number "5432" is taken or running anotheer process,
```

Comment:
Oh I see, the issue is that you have postgres SERVER running locally on your machine. In that case it would be better to use a port thats not privileged (eg > 1000). 25 would not be usable on a mac or linux for example. We could indeed use 5433 for example